### DISCUSSION TITLE
Feature request: Virtualized rows + Windows scroll

### DISCUSSION THREAD

**Discussion by meotimdihia** [TanStack/table#4318] [Ideas]
The feature will improve UI + UX and it feel like a native table.
From my experience, this is an important feature. But it seems hard to develop, I don't see any table JS supporting this feature.

**Comment by KevinVandy** [comment]
We already have an example of this with `react-virtual` https://tanstack.com/table/v8/docs/examples/react/virtualized-rows

**Reply by meotimdihia** [reply]
@KevinVandy I mean window as the scrollElement: https://tanstack.com/virtual/v3/docs/adapters/react-virtual#usewindowvirtualizer
